docs/integration-tests.md

```diff
@@ -0,0 +1,168 @@
+    1.  Add the username as `localstack-testing`.
+    2.  Keep the **Provide user access to the AWS Management Console - optional** box unchecked.
+4.  Attach existing policies directly.
+5.  Check **AdministratorAccess** and click **Next** before **Next/Create User** until done.
+6.  Go to the newly created user under `IAM/Users`, go to the `Security Credentials` tab, and click on **Create access key** within the `Access Keys` section.
+7.  Pick the **Local code** option and check the **I understand the above recommendation and want to proceed to create an access key** box.
+8.  Click on **Create access key** and copy the Access Key ID and the Secret access key immediately.
+9.  Run `aws configure —-profile ls-sandbox` and enter the Access Key ID, and the Secret access key when prompted.
+10.  Verify that the profile is set up correctly by running: `aws sts get-caller-identity --profile ls-sandbox`.
+
+Here is how `~/.aws/credentials` should look like:
+
+```bash
+[ls-sandbox]
+aws_access_key_id = <your-key-id>
+aws_secret_access_key = <your-secret-key>
+```
+
+The `~/.aws/config` file should look like:
+
+```bash
+[ls-sandbox]
+region=eu-central-1
+# .... you can add additional configuration options for AWS clients here
+```
+
+#### Running integration tests against AWS
+
+-   Set the environment variable: `TEST_TARGET=AWS_CLOUD`.
+-   Use the client `fixtures` and other fixtures for resource creation instead of methods from `aws_stack.py`
+    -   While using the environment variable `TEST_TARGET=AWS_CLOUD`, the boto client will be automatically configured to target AWS instead of LocalStack.
+-   Configure your AWS profile/credentials:
+    -   When running the test, set the environment variable `AWS_PROFILE` to the profile name you chose in the previous step. Example: `AWS_PROFILE=ls-sandbox`
+-   Ensure that all resources are cleaned up even when the test fails and even when other fixture cleanup operations fail!
+-   Testing against AWS might require additional roles and policies.
+
+Here is how a useful environment configuration for testing against AWS could look like:
+
+```bash
+DEBUG=1;  # enables debug logging
+TEST_DISABLE_RETRIES_AND_TIMEOUTS=1;
+TEST_TARGET=AWS_CLOUD;
+AWS_DEFAULT_REGION=us-east-1;
+AWS_PROFILE=ls-sandbox
+```
+
+Once you're confident your test is reliably working against AWS you can add the pytest marker `@markers.aws.validated`.
+
+#### Create a snapshot test
+
+Once you verified that your test is running against AWS, you can record snapshots for the test run. A snapshot records the response from AWS and can be later on used to compare the response of LocalStack. 
+
+Snapshot tests helps to increase the parity with AWS and to raise the confidence in the service implementations. Therefore, snapshot tests are preferred over normal integrations tests. 
+
+Please check our subsequent guide on [Parity Testing](parity-testing) for a detailed explanation on how to write AWS validated snapshot tests.
```

Comment:
I am pretty sure this link doesn't work as it should have the file ending (but I might be wrong?)...